spec/controller/scoreboard_partial_spec.rb

```diff
@@ -0,0 +1,2 @@
+require 'rails_helper' do
```

Comment:
Sorry this is part of a separate branch that we are working on. However, it leaked into our master. I will be sure to delete or just leave it blank. We will get to creating all the controller tests in one future pull request.